README.md

```diff
@@ -9,7 +9,7 @@ Supported by the Open Food Foundation and a network of global affiliates, we are
 
 We're part of global movement - get involved!
 
-* Join the conversation [on Slack][slack-invite]. Make sure you introduce yourself in the #general channel
+* Join the conversation [on Slack](https://join.slack.com/t/openfoodnetwork/shared_invite/enQtMzU2Mjk5MDc2MjA5LTM4ZTAzZjIwNzIxMmU5ODFiNWY1MTU2ZWUyNzQwNjdjNTY0N2VhY2UwOGU4ZmVjNzYyZDU2NjY3NzZkZmQwYjk). Make sure you introduce yourself in the #general channel
```

Comment:
I wonder if you should edit the `slack-invite` reference at the bottom rather than the link directly? 🙂

Reply:
Right. Check reference-style at https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Right, thx Sigmund, I completely missed that one! I've added a commit to fix this